sci-libs/opencascade/opencascade-0.18.3.recipe

```diff
@@ -0,0 +1,200 @@
+SUMMARY="A C++ 3D modeling library"
+DESCRIPTION="Development platform for CAD/CAE, 3D surface/solid modeling and \
+data exchange"
```

Comment:
Better description from the [homepage]

> Open CASCADE Technology (OCCT) is an object-oriented C++ class library designed for rapid production of sophisticated domain-specific CAD/CAM/CAE applications.

[homepage]: https://www.opencascade.com/content/overview